Sources/Event Logging/TracksLogging.m

```diff
@@ -2,9 +2,9 @@
 #import "TracksLoggingPrivate.h"
 
 DDLogLevel TracksGetLoggingLevel(void) {
-    return ddLogLevel;
+    return tracks_ddLogLevel;
```

Comment:
`ddLogLevel` is the default that will be used by all Cocoapods installations. If a user of this library already has defined a symbol called `ddLogLevel`, then we'll get duplicate symbols. Since this one is only intended for use within tracks, I'm prefixing it here.

(I ran into this when trying to integrate this into Day One.)

Reply:
Thanks for dealing with this – some history – this is a bit of an ugly workaround we had to make to sync up log levels because all our dependencies use CocoaLumberjack directly.

I've long considered using a protocol that the parent app can confirm to, which would eliminate the need for a dependency on CocoaLumberjack (and really, any specific logging framework) – instead, the parent app would delegate messages to its logging framework of choice.

Not sure if that's in-scope for this, but because we're changing so much already I wonder about trying to sneak it in 😅

Reply:
I actually thought about doing something like that, but I've been trying to keep my changes here source-compatible with other existing clients. It sounds like this change would cause Tracks logging to not appear in the logs of existing apps, so that would be a problem.

A simple logging protocol could look something like this:

```swift
@objc protocol TracksLogging {
  func logError(_: String)
  func logWarn(_: String)
  func logInfo(_: String)
  func logDebug(_: String)
  func logVerbose(_: String)
}
```

Does something like that seem reasonable?